server/auth_function/test/login_event.json

```diff
@@ -16,15 +16,15 @@
             "cognito:user_status": "EXTERNAL_PROVIDER",
             "custom:idp_display_name": "Gustafson, Conrad CITZ:IN",
             "custom:idp_name": "idir",
-            "custom:idp_user_id": "B5ECDB094DFB4149A6A8445A0MANGLED",
+            "custom:idp_user_id": "B5ECDB094DFB4149A6A8445AMANGLED0",
```

Comment:
make this guid different from the one we use for backend test, so we won't get conflict on user_guid